npm/assemble/rules.bzl

```diff
@@ -0,0 +1,83 @@
+#
```

Comment:
File containing definition of `assemble_npm` rule only (note that this is now less cluttered than before when it had 2 rule implementations in the same file, neither of which was trivial)